docs/src/ring_interface.md

```diff
@@ -505,6 +505,8 @@ throwing an exception, returning meaningless results, hanging or crashing. The
 function should only be called with `check=false` if it is already known that the
 division will be exact.
 
+***Note:*** For the *recursive* ring tests to work, one needs to also implement `divides(f::MyElem, g::MyElem)`. 
```

Comment:
I think `divides` is only required for *euclidean* rings: only `test_EuclideanRing_interface` calls it. Which is in turn only invoked automatically for univariate polynomial rings over a field...

Reply:
And there are generic `divides` methods for that, i.e.:
```
divides(f::PolyRingElem{T}, g::PolyRingElem{T}) where T <: RingElement
divides(z::PolyRingElem{T}, x::T) where T <: RingElement
```
So it would be really good to know what the actual problem you run into was?